it/src/main/java/org/corfudb/universe/node/client/ClientParams.java

```diff
@@ -0,0 +1,42 @@
+package org.corfudb.universe.node.client;
+
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import org.corfudb.universe.node.Node;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.universe.node.Node'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)